Detectors/Passive/src/Hall.cxx

```diff
@@ -86,20 +87,41 @@ void Hall::createMaterials()
   // only media needed for geometry are created
 
   //  Stainless Steel
-  matmgr.Mixture("HALL", 50, "STAINLESS STEEL3", asteel, zsteel, 7.88, 4, wsteel);
-  matmgr.Medium("HALL", 50, "STST_C2", 50, 0, isxfld, sxmgmx, tmaxfd, stemax, deemax, epsil, stmin);
+  matmgr.Mixture("HALL", kSTST_C2, "STAINLESS STEEL3", asteel, zsteel, 7.88, 4, wsteel);
+  matmgr.Medium("HALL", kSTST_C2, "STST_C2", 50, 0, isxfld, sxmgmx, tmaxfd, stemax, deemax, epsil, stmin);
```

Comment:
I see `50` also as 4th parameter, should that also be replaced by the enum value?

Reply:
The second number connects the medium to the material. If same IDs are used (as it is the case here), this second number can also be replaced by the same enum value.